scipy/special/tests/test_softmax.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import division, print_function, absolute_import
```

Comment:
It's be nice if this could be merged into `test_logsumexp.py`, since the file is the same, the testing file should be the same.

It's nice if the tests mirror the structure of the actual code.

Reply:
Agreed.  I was *this close* to making this change earlier anyway.

Reply:
Done.